domain-models-runtime/src/main/java/org/folio/rest/persist/PostgresClient.java

```diff
@@ -497,6 +502,27 @@ private void init() throws Exception {
     }
 
     client = createPgPool(vertx, postgreSQLClientConfig);
+
+    activeConnections = new CopyOnWriteArraySet<>();
```

Comment:
This overwrites static `activeConnections` when a second PostgresClient is created.